docs/environment.yml

```diff
@@ -10,6 +9,8 @@ dependencies:
   - nbconvert
   - ipykernel
   - python-dateutil
-  - nansat
```

Comment:
nansat must be installed to run Jupyter notebook tutorials

Reply:
Of course. I will fix.